src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/DbReference.scala

```diff
@@ -46,13 +45,9 @@ object DbReference extends LazyLogging {
     }
   }
 
-  def init(config: Config, system: ActorSystem)(implicit executionContext: ExecutionContext): DbReference = {
+  def init(config: Config)(implicit executionContext: ExecutionContext): DbReference = {
     val dbConfig = DatabaseConfig.forConfig[JdbcProfile]("slick", config)
 
-    system.registerOnTermination {
```

Comment:
This was the only reason `DbReference` needed an `ActorSystem` - I just moved it to `Boot.scala`.